src/Makefile

```diff
@@ -40,6 +40,7 @@ TARGETS = libpmem libvmem libpmemblk libpmemlog libpmemobj libpmempool\
 ALL_TARGETS = $(TARGETS) common librpmem
 
 SCOPE_DIRS = $(TARGETS) common librpmem rpmem_common
+POSSIBLE_TARGETS = $(SCOPE_DIRS) examples benchmarks
```

Comment:
POSSIBLE_TARGETS and SCOPE_DIRS should not be related; SCOPE_DIRS is used for cscope